### ISSUE TITLE
Zellij Freezes on Start

### ISSUE THREAD

**Issue by RaymondLChastain** [zellij-org/zellij#3059]
**Basic information**

`zellij --version`: 0.39.2

`stty size`: any

`uname -av` or `ver`(Windows):  Linux nixos 6.1.69 #1-NixOS SMP PREEMPT_DYNAMIC Wed Dec 20 16:00:29 UTC 2023 x86_64 GNU/Linux

## Further information
Reproduction steps, noticeable behavior, related issues, etc

# 2. Issues with the Zellij UI / behavior / crash

- Uninstalling and reinstalling, removing cache directories, killing all sessions, and rebooting have not helped.

## Issue description

Zellij freezes on start, a terminal is started but it just hangs, panes can't be clicked on or interacted with.

Configuration is the default.

## Minimal reproduction

Occurs any time I launch Zellij

## Other relevant information

zellij.log:
```
ERROR  |zellij::sessions         | 2024-01-13 00:29:35.410 [main      ] [src/sessions.rs:97]: Failed to read session_info cache folder: ""/home/super/.cache/zellij/0.39.2/session_info"": Os { code: 2, kind: NotFound, message: "No such file or directory" }
ERROR  |zellij::sessions         | 2024-01-13 00:29:35.410 [main      ] [src/sessions.rs:97]: Failed to read session_info cache folder: ""/home/super/.cache/zellij/0.39.2/session_info"": Os { code: 2, kind: NotFound, message: "No such file or directory" }
INFO   |zellij_client            | 2024-01-13 00:29:35.410 [main      ] [zellij-client/src/lib.rs:159]: Starting Zellij client!
INFO   |zellij_server            | 2024-01-13 00:29:35.415 [main      ] [zellij-server/src/lib.rs:246]: Starting Zellij server!
INFO   |zellij_server            | 2024-01-13 00:29:35.466 [main      ] [zellij-server/src/lib.rs:919]: Compiling plugins using Cranelift
ERROR  |zellij_client::stdin_ansi| 2024-01-13 00:29:35.466 [stdin_handler] [zellij-client/src/stdin_ansi_parser.rs:124]: Failed to open STDIN cache file: Os { code: 2, kind: NotFound, message: "No such file or directory" }
INFO   |zellij_server::plugins   | 2024-01-13 00:29:35.467 [wasm      ] [zellij-server/src/plugins/mod.rs:149]: Wasm main thread starts
ERROR  |???                      | 2024-01-13 00:29:35.470 [pty_writer] [zellij-server/src/os_input_output.rs:564]: a non-fatal error occured

Caused by:
    0: failed to set terminal id 0 to size (32, 91)
    1: failed to find terminal fd for id 0
INFO   |zellij_server::plugins::p| 2024-01-13 00:29:35.639 [async-std/runti] [zellij-server/src/plugins/plugin_loader.rs:536]: Compiled plugin 'tab-bar' in 167.5592ms
INFO   |zellij_server            | 2024-01-13 00:29:35.639 [async-std/runti] [zellij-server/src/lib.rs:919]: Compiling plugins using Cranelift
ERROR  |???                      | 2024-01-13 00:29:35.641 [pty_writer] [zellij-server/src/os_input_output.rs:564]: a non-fatal error occured

Caused by:
    0: failed to set terminal id 0 to size (32, 91)
    1: failed to find terminal fd for id 0
INFO   |zellij_server::plugins::p| 2024-01-13 00:29:35.821 [async-std/runti] [zellij-server/src/plugins/plugin_loader.rs:536]: Compiled plugin 'status-bar' in 191.326659ms
INFO   |zellij_server            | 2024-01-13 00:29:35.821 [async-std/runti] [zellij-server/src/lib.rs:919]: Compiling plugins using Cranelift
ERROR  |???                      | 2024-01-13 00:29:35.822 [pty_writer] [zellij-server/src/os_input_output.rs:564]: a non-fatal error occured

Caused by:
    0: failed to set terminal id 0 to size (32, 91)
    1: failed to find terminal fd for id 0
INFO   |zellij_client            | 2024-01-13 00:29:40.519 [main      ] [zellij-client/src/lib.rs:509]: Session detached
INFO   |zellij_client            | 2024-01-13 00:29:42.849 [main      ] [zellij-client/src/lib.rs:159]: Starting Zellij client!
ERROR  |???                      | 2024-01-13 00:29:42.849 [pty_writer] [zellij-server/src/os_input_output.rs:564]: a non-fatal error occured

Caused by:
    0: failed to set terminal id 0 to size (32, 91)
    1: failed to find terminal fd for id 0
INFO   |zellij_server            | 2024-01-13 00:29:42.849 [wasm      ] [zellij-server/src/lib.rs:919]: Compiling plugins using Cranelift
INFO   |zellij_server            | 2024-01-13 00:29:42.850 [wasm      ] [zellij-server/src/lib.rs:919]: Compiling plugins using Cranelift
ERROR  |???                      | 2024-01-13 00:29:42.850 [pty_writer] [zellij-server/src/os_input_output.rs:564]: a non-fatal error occured

Caused by:
    0: failed to set terminal id 0 to size (32, 91)
    1: failed to find terminal fd for id 0
ERROR  |???                      | 2024-01-13 00:29:42.850 [pty_writer] [zellij-server/src/os_input_output.rs:564]: a non-fatal error occured

Caused by:
    0: failed to set terminal id 0 to size (32, 91)
    1: failed to find terminal fd for id 0
INFO   |zellij_client            | 2024-01-13 00:29:44.336 [main      ] [zellij-client/src/lib.rs:509]: Session detached
INFO   |zellij_client            | 2024-01-13 00:31:20.931 [main      ] [zellij-client/src/lib.rs:159]: Starting Zellij client!
INFO   |zellij_server            | 2024-01-13 00:31:20.935 [main      ] [zellij-server/src/lib.rs:246]: Starting Zellij server!
INFO   |zellij_server            | 2024-01-13 00:31:20.986 [main      ] [zellij-server/src/lib.rs:919]: Compiling plugins using Cranelift
INFO   |zellij_server::plugins   | 2024-01-13 00:31:20.987 [wasm      ] [zellij-server/src/plugins/mod.rs:149]: Wasm main thread starts
ERROR  |???                      | 2024-01-13 00:31:20.990 [pty_writer] [zellij-server/src/os_input_output.rs:564]: a non-fatal error occured

Caused by:
    0: failed to set terminal id 0 to size (32, 91)
    1: failed to find terminal fd for id 0
INFO   |zellij_server::plugins::p| 2024-01-13 00:31:20.997 [async-std/runti] [zellij-server/src/plugins/plugin_loader.rs:495]: Loaded plugin 'tab-bar' from cache folder at '/home/super/.cache/zellij' in 5.128353ms
INFO   |zellij_server            | 2024-01-13 00:31:20.997 [async-std/runti] [zellij-server/src/lib.rs:919]: Compiling plugins using Cranelift
ERROR  |???                      | 2024-01-13 00:31:21.004 [pty_writer] [zellij-server/src/os_input_output.rs:564]: a non-fatal error occured

Caused by:
    0: failed to set terminal id 0 to size (32, 91)
    1: failed to find terminal fd for id 0
INFO   |zellij_server::plugins::p| 2024-01-13 00:31:21.007 [async-std/runti] [zellij-server/src/plugins/plugin_loader.rs:495]: Loaded plugin 'status-bar' from cache folder at '/home/super/.cache/zellij' in 14.500967ms
INFO   |zellij_server            | 2024-01-13 00:31:21.007 [async-std/runti] [zellij-server/src/lib.rs:919]: Compiling plugins using Cranelift
ERROR  |???                      | 2024-01-13 00:31:21.012 [pty_writer] [zellij-server/src/os_input_output.rs:564]: a non-fatal error occured

Caused by:
    0: failed to set terminal id 0 to size (32, 91)
    1: failed to find terminal fd for id 0
INFO   |zellij_client            | 2024-01-13 00:31:24.185 [main      ] [zellij-client/src/lib.rs:509]: Session detached
```

**Comment by imsnif** [comment]
Hi - I'm sorry you are experiencing this issue, but as you can imagine since this software has users then this issue needs a more accurate reproduction than what you've provided. Please try to troubleshoot this further, find out why this is particular to your situation - what's different about your machine or setup and then we can try to help you out.

If you need help, you're more than welcome to hop by our Discord or Matrix and ask for ideas or assistance. Until then, I'm closing this issue. Will re-open if we find an actual bug in this software.

**Comment by imsnif** [comment]
A cursory look at the logs btw says that Zellij cannot find the cache folder. This could be a permission issue of some kind on your system. I'd start in that direction.

**Comment by RaymondLChastain** [comment]
Hi @imsnif, the cache directory exists, and is fully populated as normal.  There is no reproduction to be done here, install the package on nixos it doesn't work, there is one log, these are it's contents.

You are a rude and ignorant person, I will stop using your software, and will stop using it at the companies I work for.

**Comment by RaymondLChastain** [comment]
It is impossible for users to help diagnose when your software emits useless logs with no information.

**Comment by RaymondLChastain** [comment]
Regardless, freezing with no additional helpful information, is not a good user experience. You should be grateful I even brought you your useless log.

**Comment by imsnif** [comment]
Hi @RaymondLChastain - I deleted your comments. Please refer to our [Code of Conduct](https://github.com/zellij-org/zellij/blob/main/CODE_OF_CONDUCT.md).

**Comment by RaymondLChastain** [comment]
Again, won't be doing that, please refer to human decency and respect.

For every user that reports a bug, 1000 walk away. Switching to tmux now, goodbye.

**Comment by imsnif** [comment]
Hi @RaymondLChastain - I changed the issue title back to the original. I will again refer you to our CoC. One more violation and I will have to block you indefinitely.

**Comment by RaymondLChastain** [comment]
I've already blocked you, please feel free to do the same, stop sending messages, I'm sick of seeing your name in my email.

**Comment by imsnif** [comment]
This user has been blocked from the Zellij organization.

For the sake of transparency, for others reading this: this user has repeatedly engaged in behavior in violation of our [Code of Conduct](https://github.com/zellij-org/zellij/blob/main/CODE_OF_CONDUCT.md). Specifically engaging in insulting/derogatory comments and personal attacks (against me), and engaging in conduct that would be considered inappropriate in a professional setting.  I have saved the deleted comments and will produce them if needed. 

I started by deleting the relevant comments and warning the user. But after a third violation I was forced to block them from the organization.

My heart goes out to this person and I regret having to do this, but we do not accept such behavior here.
